client/app/components/header.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import QuestionSubmissionForm from './questionSubmissionForm.jsx';
+import styles from '../styling/app.css';
+
+class Header extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      toggle: false,
+    };
+  }
+
+  displaySubmissionForm() {
+    const boolean = this.state.toggle;
+    this.setState({
+      toggle: !boolean,
```

Comment:
On an initial read this is slightly confusing, that a state property is toggle; when it's set to true does something happen automatically, like a trigger? I think the label toggle sounds like an event handler. Not a big deal though!